ui/page/dexclient/listeners.go

```diff
@@ -0,0 +1,18 @@
+package dexclient
+
+import "decred.org/dcrdex/client/core"
+
+// readNotifications reads from the Core notification channel.
+func (pg *Page) readNotifications() {
+	ch := pg.Dexc().Core().NotificationFeed()
+	for {
+		select {
+		case n := <-ch:
+			if n.Type() == core.NoteTypeFeePayment {
+				pg.RefreshWindow()
+			}
+		case <-pg.ctx.Done():
+			return
+		}
+	}
+}
```

Comment:
This was only used in the market.go file... Kindly move this to the market to go file.